pyiceberg/catalog/sql.py

```diff
@@ -110,8 +110,28 @@ def __init__(self, name: str, **properties: str):
 
         if not (uri_prop := self.properties.get("uri")):
             raise NoSuchPropertyException("SQL connection URI is required")
-        echo = bool(self.properties.get("echo", False))
-        self.engine = create_engine(uri_prop, echo=echo)
+
+        echo: Union[str, bool]
+        echo_str = str(self.properties.get("echo", "false")).lower()
```

Comment:
Should we move the defaults to a constant?

Reply:
Seems fair